cas-server-core/src/main/java/org/jasig/cas/ticket/TicketValidationException.java

```diff
@@ -24,7 +24,6 @@
  * Exception to alert that there was an error validating the ticket.
  *
  * @author Scott Battaglia
-
  * @since 3.0.0
  */
 public class TicketValidationException extends TicketException {
```

Comment:
Don't you want to make it `abstract` as the root hierarchy of the ticket validation exception?